src/dreyfus/src/clouseau_rpc.erl

```diff
@@ -24,32 +24,138 @@
 -export([set_purge_seq/2, get_purge_seq/1, get_root_dir/0]).
 -export([connected/0]).
 
+%% string represented as binary
+-type string_as_binary(_Value) :: binary().
```

Comment:
There is [nonempty_binary/0](https://www.erlang.org/doc/apps/erts/erlang#t:nonempty_binary/0) as well now if that constrains the type better.